internal/definitions/sender_contracts_test.go

```diff
@@ -306,6 +306,27 @@ func TestDefineContractAPIPublishNonMultiparty(t *testing.T) {
 	assert.Regexp(t, "FF10414", err)
 }
 
+func TestDefineContractAPIPublishNonMultipartyUpdate(t *testing.T) {
```

Comment:
This isn't a "publish" test (so I would remove the word "publish" from the name).